template.yaml

```diff
@@ -458,8 +458,241 @@ Resources:
       MessageRetentionPeriod: 1209600
       KmsMasterKeyId: !Ref AuditKmsKeyAlias
       Tags:
-         - Key: PersonalIdentifyingInformation
-           Value: true
+        - Key: PersonalIdentifyingInformation
+          Value: true
+
+  AlertTopic:
+    Type: AWS::SNS::Topic
+    Properties:
+      # checkov:skip=CKV_AWS_26: We do not need to encrypt this data on the SNS topic as alerts aren't sensitive
+      DisplayName: !Sub ${Environment}${Developer}-alert-topic
+      TopicName: !Sub ${Environment}${Developer}-alert-topic
+      Tags:
+        - Key: CheckovRulesToSkip
+          Value: CKV_AWS_26
+
+  AlertTopicSubscription:
+    Type: AWS::SNS::Subscription
+    Properties:
+      Endpoint: di-life-events-platform@digital.cabinet-office.gov.uk
+      Protocol: email
+      TopicArn: !Ref AlertTopic
```

Comment:
Think we might want `!Ref AlertTopic.TopicArn`

Reply:
Pretty sure in this case !Ref gets the ARN - I tested this with my personal email and it worked.